examples/react-native/__tests__/__snapshots__/Intro-test.js.snap

```diff
@@ -1,3 +1,5 @@
+// Jest Snapshot v1
```

Comment:
Let's do:

// Jest Snapshot v1, https://goo.gl/fbAQLP

which links to our snapshot testing guide.

cc @kentcdodds who will be pleased.